models/GroupsService.php

```diff
@@ -112,7 +113,8 @@ public function isGroupClass( core_kernel_classes_Class $clazz)
      */
     public function getGroups($userUri)
     {
-        return $this->getRootClass()->searchInstances(array(self::PROPERTY_MEMBERS_URI => $userUri), array('like'=>false, 'recursive' => true));
+        $user = new core_kernel_classes_Resource($userUri);
+        return $user->getPropertyValues(new core_kernel_classes_Property(self::PROPERTY_MEMBERS_URI));
```

Comment:
These specific modification change the signature of what method getGroups returned 

This PR is not valid